### PR TITLE
Create Log in page

### LINKED ISSUE #31: Create LogIn page
As a user who already has an account
I want to be able to log in
therefore a log in form is needed.
Screenshot 2021-09-13 at 13 25 02

### PR DESCRIPTION
**Pull request by aronvaupel**
Create basic markup for page.
Styling will update once SignIn-page is appproved and merged.
Same applies for further logic, which will also be implemented after approval of SignInPage

closes #31

### CODE REVIEW

File: src/app/pages/LogInPage/LogInPage.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <div className={styles.wrapper}>
+      <Header />
+      <main className={styles.mainWrapper}>
+        <div className={styles.input}>
+          <Input placeholder="Enter username" label="Username" id="username" />
+          <Input placeholder="Enter password" label="Password" id="password" />
+        </div>
+        <div className={styles.action}>
+          <ActionButton type="submit" value="Submit" onClick={handleClick} />
+        </div>
```
Comment (phlgr) [inline src/app/pages/LogInPage/LogInPage.tsx:22]: This should be wrapped in a form, also the two inputs should be controlled.

File: src/app/pages/OptionsPage/OptionsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import LogInSignInLink from '../../components/LogIn-SignIn-Link/LogIn-SignIn-Link';
+
+function OptionsPage(): JSX.Element {
+  return (
+    <div className={styles.wrapper}>
+      <Header />
+      <main className={styles.mainWrapper}>
+        <div className={styles.linkWrapper}>
+          <LogInSignInLink url="/signIn" value="Sign in" />
+          <LogInSignInLink url="/LogIn" value="Log in" />
```
Comment (phlgr) [inline src/app/pages/OptionsPage/OptionsPage.tsx:14]: Please use consistes casing.
```suggestion
          <LogInSignInLink url="/signin" value="Sign in" />
          <LogInSignInLink url="/login" value="Log in" />
```

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/pages/LogInPage/LogInPage.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  const handleSubmit = () => {
+    console.log('Click');
+  };
+  return (
+    <div className={styles.wrapper}>
+      <Header />
+      <main className={styles.mainWrapper}>
+        <form className={styles.form}>
+          {' '}
```
Comment (marwinburesch) [inline src/app/pages/LogInPage/LogInPage.tsx:26]: this can go :)

File: src/app/pages/LogInPage/LogInPage.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const validUsername = () => username.length < 3;
+  const validPassword = () => password.length < 8;
+
+  const isDisabled = validUsername() || validPassword();
+
+  const handleSubmit = () => {
+    console.log('Click');
```
Comment (marwinburesch) [inline src/app/pages/LogInPage/LogInPage.tsx:19]:
```suggestion
    console.log('Submitted');
```

**Review by marwinburesch — COMMENTED** [review]
Now the functionality :)

**Pull request merged**